tests/DCPS/Compiler/keywords/cpp11/main.cpp

```diff
@@ -0,0 +1,32 @@
+#include "testTypeSupportImpl.h"
+
+int ACE_TMAIN(int, ACE_TCHAR**)
+{
+  the_module::the_struct a;
+  a.the_field() = the_module::the_const;
+  ACE_UNUSED_ARG(a);
+  the_module::the_structDataWriter* a_dw = 0;
+  ACE_UNUSED_ARG(a_dw);
+
+  boolean::attribute b;
+  b.component() = boolean::oneway;
+  ACE_UNUSED_ARG(b);
+  boolean::attributeDataWriter* b_dw = 0;
+  ACE_UNUSED_ARG(b_dw);
+
+  /* TODO(iguessthislldo): See test.idl
+  _cxx_bool::_cxx_class c;
+  c._cxx_else() = _cxx_bool::_cxx_continue;
+  ACE_UNUSED_ARG(c);
+  _cxx_bool::cxx_classDataWriter* c_dw = 0;
```

Comment:
This is not as I would expect it, first the implied IDL rules from the DDS specification should be applied, after that the language mapping escaping on the implied IDL, so it should be `_cxx_bool::classDataWriter`.

Reply:
This was just result of applying the existing code to this, but this sounds right. I could change `EscapeContext_ForGenIdl` to strip `_cxx_` and I think it will happen that way.

Reply:
Alright done and I was able to test using the ACE/TAO PR.